orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTask.groovy

```diff
@@ -224,6 +233,20 @@ class MonitorKatoTask implements RetryableTask, CloudProviderAware {
     }
   }
 
+  /**
+   * Log and emits a metric when a kato task retry is finally terminal - typically either from
+   * the task timing out or from an unexpected error, like a 404, when attempting to retry.
+   */
+  private void monitorFinalTerminalRetry(StageExecution stage, String reason) {
```

Comment:
nit: i am not sure i "get" the name `monitorFinalTerminalRetry` maybe more like `logFinalTerminalRetry` but obv not a strong feeling here

Reply:
Ah yea, I went with `monitor` since we also emit a metric from this function.  I was toying with `observeFinalTerminalRetry` too.

Reply:
But yea the term `monitor` and `observe` can be sort of overloaded.